packages/core/src/components/slider/multiSlider.tsx

```diff
@@ -41,10 +41,11 @@ export interface ISliderBaseProps extends IProps, IIntentProps {
     disabled?: boolean;
 
     /**
-     * Increment between successive labels. Must be greater than zero.
+     * As a number, increment between successive labels. Must be greater than zero.
+     * As an array of numbers, places label that corresponds to each entry.
      * @default 1
      */
-    labelStepSize?: number;
+    labelStepSize?: number | number[];
```

Comment:
I think `labelStepSize` no longer makes sense as a name for this prop if it's going to be an array of explicit labels. Let's instead add this feature with a new prop called `labelValues: number[]` and add validation inside `validateProps` to make it mutually exclusive w/ `labelStepSize` (see [here](https://github.com/palantir/blueprint/blob/ba5c9b56e03642c8168fba55e1766fd3ec864e32/packages/core/src/components/forms/radioGroup.tsx#L87-L89) for an example of that kind of check in another component).

Reply:
So currently, `labelStepSize` defaults to 1. And if I were to add a new prop called `labelValues` with the default of `null`, and they are meant to be mutually exclusive, does that mean the user should manually set the `labelStepSize` to `null` in order to use `labelValues`?

Reply:
I guess this would be fixed by allowing the `labelValues` to override `labelStepSize` (while still throwing a warning indicating they should be mutually exclusive?). Should I use this implementation?

Reply:
a slightly different suggestion: remove the default value of `labelStepSize` in `defaultProps`, instead implement that default value dynamically in a private helper method `getLabelValues()`. then you can still validate for mutual exclusivity properly